vizro-core/src/vizro/static/css/loading.css

```diff
@@ -0,0 +1,26 @@
+/* Loading spinner on layout load - ie when the Vizro app is not shown yet */
+
+/* Inspired by https://github.com/facultyai/dash-bootstrap-components/blob/5c8f4b40f1100fc00bf2d5c1d671a7815a6b2910/docs/static/loading.css */
+._dash-loading {
+  background: rgba(20, 23, 33, 1);
+  color: transparent;
+  inset: 0;
+  margin: 0;
+  position: fixed;
+  text-align: center;
```

Comment:
Optional: Since we are giving position fixed. and not specifying any top, left etc. This div will remain at the place accroding to how the html is rendered.

If you want to fix it to the middle of the screen or to take the whole screen, you can provide those values so that you see the loaded always at a consisiten place